_partials/_deprecated.mdx

```diff
@@ -0,0 +1,8 @@
+<highlight type="deprecation">
+This section describes a feature that is deprecated on TimescaleDB. We strongly
+recommend that you do not use this feature in a production environment. For some
+suggestions of workarounds, see this
+[Timescale Forum post](https://www.timescale.com/forum/t/telegraf-plugin/118).
```

Comment:
**suggestion (non-blocking):** Maybe take the "For some suggestions of workarounds..." out and put it under the highlight? It's not going to be as visible, but the partial will be more reusable.